docs/src/jsx/LinesInstability.js

```diff
@@ -6,33 +6,32 @@
 
 import React from "react";
 
-import { p } from "./utils";
 import Worldview, { Lines, DEFAULT_CAMERA_STATE } from "regl-worldview";
 
 // #BEGIN EXAMPLE
 function LinesInstabilityDemo() {
   const points = [
-    { x: -812.2277333190451, y: 2961.4633761946707, z: 0 },
-    { x: -812.2718382693613, y: 2960.8755785794347, z: 0 },
-    { x: -812.3047227216128, y: 2960.4388610900487, z: 0 },
-    { x: -812.3249921796464, y: 2960.128621087491, z: 0 },
-    { x: -812.3386504915552, y: 2959.9166965937397, z: 0 },
-    { x: -812.3474835309406, y: 2959.779641779605, z: 0 },
-    { x: -812.3526775112591, y: 2959.699051172442, z: 0 },
-    { x: -812.3552507121985, y: 2959.6591249861, z: 0 },
-    { x: -812.3561798454938, y: 2959.644708409252, z: 0 },
-    { x: -812.3563502479498, y: 2959.6420644182203, z: 0 },
-    { x: -812.3563594592313, y: 2959.6419214945413, z: 0 },
-    { x: -812.3563605869005, y: 2959.641903997451, z: 0 },
-    { x: -812.3563606674282, y: 2959.6419027479683, z: 0 },
-    { x: -812.3563610277726, y: 2959.6418971568114, z: 0 },
-    { x: -812.3563612249804, y: 2959.641894096902, z: 0 },
-    { x: -812.3563613255686, y: 2959.641892536161, z: 0 },
-    { x: -812.3726781860942, y: 2959.3887175882182, z: 0 },
-    { x: -812.4113489277303, y: 2958.368466516476, z: 0 },
+    { x: 812, y: 2961, z: 0 },
+    { x: 812, y: 2960, z: 0 },
+    { x: 812, y: 2960, z: 0 },
+    { x: 812, y: 2960, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2959, z: 0 },
+    { x: 812, y: 2958, z: 0 },
```

Comment:
This whole line instability business shouldn't be a demo right? I thought it was intended as a regression test for an edge case and should probably only exist as a screenshot test. I wonder if we should touch it at all — maybe messing with these numbers makes it not test the edge case properly.

Reply:
Good point. I'll leave the demo unchanged.